### PR TITLE
docs: e2ee added to create session

### PR DESCRIPTION
**Pull request by manchuck**
#### What is this PR doing?
Adding documentation for the e2ee parameter

### CODE REVIEW

File: lib/opentok.js [hunk 1]
```diff
@@ -1815,12 +1815,17 @@ OpenTok.prototype.dial = function (sessionId, token, sipUri, options, callback)
...
  *     <li><code>location</code> (String) &mdash;
  * An IP address that the OpenTok servers will use to situate the session in the global
  * OpenTok network. If you do not set a location hint, the OpenTok servers will be based on
  * the first client connecting to the session.
  * </li>
+ * 
+ *     <li><code>e2ee</code> (String[true|false]) &mdash;
+ * This feature adds an encryption layer by encrypting the media payload at the client to 
+ * remain encrypted through the media server when routing media to other clients. You must 
+ * enable end-to-end encryption when you create a session.
```
Comment (jeffswartz) [inline lib/opentok.js:1827]:
```suggestion
 * Determines whether to enable <a href="https://tokbox.com/developer/guides/end-to-end-encryption/">
`end-to-end encryption</a>
for the OpenTok session. Set to <code>"true"</code> or <code>"false"</code>.
```

File: lib/opentok.js [hunk 2]
```diff
@@ -1815,12 +1815,17 @@ OpenTok.prototype.dial = function (sessionId, token, sipUri, options, callback)
...
  *
  * <ul>
- *
  *     <li><code>location</code> (String) &mdash;
  * An IP address that the OpenTok servers will use to situate the session in the global
  * OpenTok network. If you do not set a location hint, the OpenTok servers will be based on
  * the first client connecting to the session.
  * </li>
+ * 
+ *     <li><code>e2ee</code> (String[true|false]) &mdash;
```
Comment (jeffswartz) [inline lib/opentok.js:1824]: Do you really need to set this to a string (`'true'` or `'false'`)? Can you pass in a boolean (`true` or `false`)?

**Review by jeffswartz — CHANGES_REQUESTED** [review]

File: lib/opentok.js (hunk 2 above)
Comment (manchuck) [reply lib/opentok.js:1830]: as far as I know it has to be a string.

Comment (jeffswartz) [reply lib/opentok.js:1830]: I'm pretty sure that this is a Boolean.

File: lib/render.js [hunk 3]
```diff
@@ -204,12 +208,15 @@ exports.getRender = (config, renderId, callback) => {
 };
 
 /**
- * Starts an Experience composer for an OpenTok session.
+ * Starts an Experience cCmposer for an OpenTok session.
```
Comment (jeffswartz) [inline lib/render.js:211]:
```suggestion
 * Starts an Experience Composer for an OpenTok session.
```

**Review by jeffswartz — DISMISSED** [review]

File: lib/render.js [hunk 4]
```diff
@@ -204,12 +208,15 @@ exports.getRender = (config, renderId, callback) => {
...
- * Starts an Experience composer for an OpenTok session.
+ * Starts an Experience Composer for an OpenTok session.
  *
  *
  * @param {Object} config - API configuration settings {@see OpenTok.startRender}
  * @param {Object} options - Optional parameters for the API call {@see OpenTok.startRender}
- * @param {Function} callback - Call back function
+ * @param {Function} callback - Callback function
+ * 
+ * @method #startRebder
```
Comment (jeffswartz) [inline lib/render.js:218]:
```suggestion
 * @method #startRender
```

**Review by jeffswartz — APPROVED** [review]

**Pull request merged**